apps/calc/src/components/graphic/line.tsx

```diff
@@ -1,16 +1,28 @@
 import { Limits, Point2D } from "@arithmico/engine/lib/types";
 import classNames from "classnames";
-import { convertToViewPortCoordinates } from "./graphic";
+import { convertToViewPortCoordinates } from "@local-components/graphic/graphic-utils";
 
 interface LineProps {
   points: Point2D[];
   limits: Limits;
+  viewBoxWidth: number;
```

Comment:
vielleicht kannst du viewBoxWidth und viewBoxHeight in einer Struktur wie viewBoxDimensions zusammenfassen, da die attribute nie einzeln vorkommen